src/main/scala/io/iohk/ethereum/db/storage/pruning/RangePrune.scala

```diff
@@ -0,0 +1,15 @@
+package io.iohk.ethereum.db.storage.pruning
+
+import io.iohk.ethereum.utils.Logger
+
+trait RangePrune extends Logger {
+  def pruneBetween(start: BigInt, end: BigInt, pruneFn: BigInt => Int): PruneResult = {
```

Comment:
what's the difference between pruneFn and PruneFn from PruningMode? Can't we have a single pruneFn (with the same signature?)